docs/SortOrder.md

```diff
@@ -0,0 +1,7 @@
+# SquareConnect::SortOrder
```

Comment:
Yes. it's worth to investigate. Th python doc has its content: https://github.com/square/connect-python-sdk/blob/release/2.0.2/docs/SortOrder.md